zuul-core/src/test/java/com/netflix/zuul/context/DebugTest.java

```diff
@@ -59,7 +60,7 @@ public void setup() {
         params.add("k1", "v1");
 
         request = new HttpRequestMessageImpl(ctx, "HTTP/1.1", "post", "/some/where",
-            params, headers, "9.9.9.9", "https", 80, "localhost");
+            params, headers, "9.9.9.9", "https", 80, "localhost",new InetSocketAddress("api.netflix.com", 443));
```

Comment:
This is not ideal, because it will do a DNS lookup as part of the test, making it possibly more flaky, and slower.